### PR TITLE
Update README to us 8.0 instead of 1.7

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Update README to us 8.0 instead of 1.7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,20 +6,20 @@
 ## Supported tags
 
 ### Images by PrestaShop version
-* `1.7`, `latest`, `1.7-apache`
-* `1.7-fpm`
+* `8.0`, `latest`, `8.0-apache`
```
[the remark this answers is not in the document]
Comment (jolelievre) [reply README.md:9]: They both exist anyway https://hub.docker.com/layers/prestashop/prestashop/8-apache/images/sha256-c500df464ac4409f8e5d221d5efce0d9a6b90f4c95cfb8850176c4dfc86ac4cc?context=explore
https://hub.docker.com/layers/prestashop/prestashop/8.0-apache/images/sha256-c500df464ac4409f8e5d221d5efce0d9a6b90f4c95cfb8850176c4dfc86ac4cc?context=explore

But if we don't want to edit this again for 8.1 maybe 8 is better

[the remark this answers is not in the document]
Comment (matks) [reply README.md:9]:
```suggestion
* `8`, `latest`, `8-apache`
```

File: README.md [hunk 2]
```diff
@@ -6,20 +6,20 @@
 ## Supported tags
 
 ### Images by PrestaShop version
-* `1.7`, `latest`, `1.7-apache`
-* `1.7-fpm`
+* `8`, `latest`, `8-apache`
+* `8.0-fpm`
```
Comment (matks) [inline README.md:10]:
```suggestion
* `8-fpm`
```

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (matks) [reply README.md]: Fixed

**Review by nicosomb — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Review by okom3pom — APPROVED** [review]

**Comment by matthieu-rolland** [comment]
thank you @matks

**Pull request merged**